src/ServiceV2/ResettingService.php

```diff
@@ -108,7 +108,7 @@ public function isRequestingBySMS(?User $user): bool
         return 0 < count($requests) && null !== $requests[0]->getSmsCode();
     }
 
-    public function processSendingPasswordResetEmail(string $email, string $locale): void
+    public function processSendingPasswordResetEmail(string $email): void
```

Comment:
I'm not a fan of this method name. `sendPasswordResetEmail` would be better in my opinion. 
We should avoid using the word process in methods names as much as possible as it is way too generic and gives so little information about what the method is actually doing.